packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -429,6 +448,16 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
                 : {}));
   }
 
+  Map<String, UpdateUserAttributeResult> _formatUpdateUserAttributesResponse(
+      Map<String, dynamic> res) {
+    return res.map((key, value) => MapEntry(
+        key,
+        _formatUpdateUserAttributeResponse(
+            new Map<String, dynamic>.from(value))));
```

Comment:
Nit: Think we should omit the `new` keyword (https://dart.dev/guides/language/effective-dart/usage#dont-use-new)
```suggestion
           Map<String, dynamic>.from(value))));
```